Detectors/TRD/macros/convertRun2ToRun3Digits.C

```diff
@@ -0,0 +1,139 @@
+#if !defined(__CINT__) || defined(__MAKECINT__)
+#include <TClonesArray.h>
+// #include <TEveLine.h>
+// #include <TEveManager.h>
+// #include <TEveElement.h>
+
+#include <AliRunLoader.h>
+#include <AliLoader.h>
+#include <AliDataLoader.h>
+#include <AliTreeLoader.h>
+#include <AliTRDarrayADC.h>
+// #include <AliTRDtrackletWord.h>
+// #include <AliTRDtrackletMCM.h>
+
+#include <iostream>
+
+#include "TH1F.h"
+// #include "AliTRDtrackletMCM.h"
+//#include "TRDDataFormat/TriggerRecord.h"
+#include "TRDBase/Digit.h"
+// #include "TRDBase/Tracklet.h"
+#endif
+
+using namespace o2;
+using namespace trd;
+using namespace std;
+
+
+void convertRun2ToRun3Digits(TString infilename = "TRD.Digits.root",
+              TString outfilename = "digitsqa.root")
```

Comment:
I would add the name of the digits output file as a command line argument to make it more readable. 

The QA could be removed, but it might be good to keep it in. Maybe you can make it optional: do not create the QA file if the filename is empty.

Reply:
And I think O2 usually produces `trddigits.root`. It might be more convenient for the use to default to this output file name.